tensorflow/python/framework/ops.py

```diff
@@ -44,6 +44,7 @@
 from tensorflow.python.platform import tf_logging as logging
 from tensorflow.python.util import compat
 from tensorflow.python.util import decorator_utils
+from tensorflow.contrib.framework import deprecated
```

Comment:
This causes several test failures on Mac:
==================== Test output for //tensorflow/tensorboard/lib/python:http_test:
Traceback (most recent call last):
  File "/private/var/tmp/_bazel_jenkins/13e370a18c169b19baeafefb05212b85/execroot/tensorflow-pull-requests-mac/bazel-out/local-opt/bin/tensorflow/tensorboard/lib/python/http_test.runfiles/org_tensorflow/tensorflow/tensorboard/lib/python/http_test.py", line 27, in <module>
    from tensorflow.python.platform import test
  File "/private/var/tmp/_bazel_jenkins/13e370a18c169b19baeafefb05212b85/execroot/tensorflow-pull-requests-mac/bazel-out/local-opt/bin/tensorflow/tensorboard/lib/python/http_test.runfiles/org_tensorflow/tensorflow/python/platform/test.py", line 67, in <module>
    from tensorflow.python.framework import test_util as _test_util
  File "/private/var/tmp/_bazel_jenkins/13e370a18c169b19baeafefb05212b85/execroot/tensorflow-pull-requests-mac/bazel-out/local-opt/bin/tensorflow/tensorboard/lib/python/http_test.runfiles/org_tensorflow/tensorflow/python/framework/test_util.py", line 37, in <module>
    from tensorflow.python.client import session
  File "/private/var/tmp/_bazel_jenkins/13e370a18c169b19baeafefb05212b85/execroot/tensorflow-pull-requests-mac/bazel-out/local-opt/bin/tensorflow/tensorboard/lib/python/http_test.runfiles/org_tensorflow/tensorflow/python/client/session.py", line 30, in <module>
    from tensorflow.python.framework import ops
  File "/private/var/tmp/_bazel_jenkins/13e370a18c169b19baeafefb05212b85/execroot/tensorflow-pull-requests-mac/bazel-out/local-opt/bin/tensorflow/tensorboard/lib/python/http_test.runfiles/org_tensorflow/tensorflow/python/framework/ops.py", line 47, in <module>
    from tensorflow.contrib.framework import deprecated
ImportError: No module named contrib.framework